### PR TITLE
hunter.io fixed

### PR DESCRIPTION
**Pull request by rachejazz**
[asciicast](https://asciinema.org/a/o0h8aZ6ExzSynciyywSUDHsRy)

PR #71 for v2 done

### CODE REVIEW

File: modules/osint/email_search.py [hunk 1]
```diff
@@ -35,12 +36,20 @@
...
+	eng = name
 	name = engine.__init__.__name__
 	varnames = engine.__init__.__code__.co_varnames
-	if name == 'ask':
-		q = q.replace('"', '')
+	q = q_formats[f"{eng}"] if f"{eng}" in q_formats else q_formats['default']
 	if 'limit' in varnames and 'count' in varnames:
 		attr = engine(q, limit, count)
+	elif 'limit' in varnames and 'key' in varnames:
+		key = q.split("&api_key=")[1]
```
Comment (saeeddhqan) [inline modules/osint/email_search.py]: ' instead of "

File: modules/osint/email_search.py [hunk 2]
```diff
@@ -35,12 +36,20 @@
...
+		k_q = q.split("&api_key=")[0]
```
Comment (saeeddhqan) [inline modules/osint/email_search.py]: ' instead of "

File: modules/osint/email_search.py [hunk 3]
```diff
@@ -35,12 +36,20 @@
...
+		if key == "None":
```
Comment (saeeddhqan) [inline modules/osint/email_search.py]: ' instead of "

**Review by saeeddhqan — COMMENTED** [review]
13 files changed. it just needs two files.

**Comment by rachejazz** [comment]
> 13 files changed. it just needs two files.

Yeah, I commited the changes with the commented lines. So sorry. I have deleted all those comments. So they are unchanged

File: modules/osint/email_search.py (hunk 3 above)
Comment (rachejazz) [reply modules/osint/email_search.py]: ok! sending the corrected one

**Pull request merged**